keyboards/sol/eeprom-righthand.eep

```diff
@@ -0,0 +1,2 @@
+:0F000000000000000000000000000000000000F1
```

Comment:
These files also reside in the lets split folder, and in /quantum/split_common.  It doesn't need to be re-uploaded here.